templates/user/heatmap.tmpl

```diff
@@ -1,5 +1,12 @@
 {{if .HeatmapData}}
-	<div id="user-heatmap" data-heatmap-data="{{Json .HeatmapData}}">
+	<div id="user-heatmap"
+		data-heatmap-data="{{Json .HeatmapData}}"
+		data-locale-total-contributions-html="{{$.locale.Tr "heatmap.number_of_contributions_in_the_last_12_months" (LocaleNumber .HeatmapTotalContributions)}}"
```

Comment:
This one could wait for #24134, then no need to use the `v-html` hack.

Reply:
Agreed